src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -46,6 +46,7 @@ class Request
     const HEADER_X_FORWARDED_PORT = 0b10000;
     const HEADER_X_FORWARDED_ALL = 0b11110; // All "X-Forwarded-*" headers
     const HEADER_X_FORWARDED_AWS_ELB = 0b11010; // AWS ELB doesn't send X-Forwarded-Host
+    const HEADER_X_FORWARDED_PREFIX = 0b11011;
```

Comment:
this value is wrong. `X_FORWARDED_PREFIX` is a separate header, so it should use a separate bit in the bit mask, not a value equal to `HEADER_X_FORWARDED_AWS_ELB | HEADER_FORWARDED`

Reply:
Thanks for that hint! In that case we must change the bitmask length, right? I have adjusted the value to `HEADER_X_FORWARDED_PREFIX=0b100000`.